Public/Test-AuthenticodeSignature.ps1

```
@@ -21,5 +21,15 @@ function Test-AuthenticodeSignature {
         [string] $Path
     )
 
+    <# TODO: PowerShell Core 6.0 does not support Get-AuthenticodeSignature
+    https://docs.microsoft.com/en-us/powershell/scripting/whats-new/known-issues-ps6?view=powershell-6
+    
+    For now, just return false as we can't test the signature, but Microsoft say this should be fixed
+    in a future release.
+    #>
+    if ($PSVersionTable.Platform -and $PSVersionTable.Platform -eq 'Unix') {
+        return $false
+    }
+
```

It would probably be better to raise an error on non-Windows platforms, rather than lie. This would give a better indication to consumers that they can't use this functionality on non-Windows platforms.

---

Thinking about it, it's not so much platform specific but Powershell version specific. Powershell Core doesn't have the cmdlet.

---

Okay, so a different test and error message. But definitely raise an error rather than mislead.

You can evaluate the following to see if the command exists:

```PowerShell
[bool] (Get-Command -Name 'Get-AuthenticodeSignature' -ErrorAction 'SilentlyContinue')
```